source/index.html.erb

```diff
@@ -46,6 +46,26 @@ per_page: 4
   </div>
 </div>
 
+<div class="featured-tip">
+  <div class="container">
+    <% blog.articles.select{ |article| article.data.category == 'tips' }[0...1].each do |article| %>
+      <div class="left">
+        <h4><%= article.data.pre_title %></h4>
+        <h2><%= article.data.title %></h2>
+        <%= article.summary %>
+        <a class="link-light" href="<%= article.url %>">Get More Tips</a>
+      </div>
+      <div class="right">
+        <img src="<%= article.data.illustration_tag %>" class="fadeIn" />
+      </div>
+    <% end %>
+  </div>
+</div>
+
+<div class="drawn-rule">
```

Comment:
Perhaps use the `hr` tag? It seems more semantic.

http://htmlreference.io/element/hr/